packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -370,8 +360,8 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           url = wrapId(resolved.id)
         }
 
-        // make the URL browser-valid if not SSR
-        if (!ssr) {
+        // make the URL browser-valid
```

Comment:
I wonder whether the code becomes simpler if we move these browser-specific code to post-process like we do in SSR ([`ssrTransform` function](https://github.com/vitejs/vite/blob/e961b31493f8493277b46773156cc6e546b9c86b/packages/vite/src/node/ssr/ssrTransform.ts#L44-L55)).  (I have not thought about it deeply yet though.)

Reply:
Probably good to keep the diff as is for now. Maybe you could try a PR after we merge this?

Reply:
Yeah, I agree a separate PR would be better👍I'll try it later on.